### PR TITLE
Issue/1824

### PR DESCRIPTION
**Pull request by mehul0810**
## Description
This PR is for #1824 

## How Has This Been Tested?
Tested the link for donation column to redirect to donation listing with respective form filter on.
Also, tested changing the form dropdown on donations list page.

@DevinWalker  @ravinderk However, I do have a suggestion. If we can remove the `Apply` button for date range filter and can have a common button named `Filter` for date range as well as forms dropdown? Let me know your thoughts.
**Screenshot:** https://tppr.me/Rj5wX

## Types of changes
Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code has proper inline documentation.

### CODE REVIEW

**Comment by ravinderk** [comment]
@mehul0810 Two suggestions:
1. Upload screenshot directly to issue.
2. We will cover your suggestion into the new search API development.
Check details here: https://docs.google.com/document/d/1bXYKDgkfNDYVj1yyVlQXVU8wt5bfjWiSccLRnZZJQ4o/edit

**Comment by mehul0810** [comment]
@ravinderk 
1. Facing issue with uploading screenshots generated in PNG format using tools. So, i've attached link for now.
2. I'll check the Search API document. Thanks!

File: includes/payments/functions.php [hunk 1]
```diff
@@ -480,7 +480,11 @@ function give_count_payments( $args = array() ) {
 	}// End if().
 
 	if ( ! empty( $args['form_id'] ) && is_numeric( $args['form_id'] ) ) {
```
Comment (ravinderk) [inline includes/payments/functions.php]: @DevinWalker Check this. It was bug in our code  :)

**Comment by DevinWalker** [comment]
@mehul0810 can you fix the merge conflict here?

**Comment by mehul0810** [comment]
@DevinWalker Solved Merge Conflict and added required changes after testing.

**Comment by DevinWalker** [comment]
@mehul0810 thanks

**Pull request merged**